integrations/server/test_covidcast_endpoints.py

```diff
@@ -530,19 +530,51 @@ def test_correlation(self):
     def test_csv(self):
         """Request a signal the /csv endpoint."""
 
+        expected_columns = ["geo_value", "signal", "time_value", "issue", "lag", "value", "stderr", "sample_size", "geo_type", "data_source"]
```

Comment:
it's reflecting the same arbitrary order specified in `parse_csv_row` ¯\(ツ)/¯

i didn't know if there was a reason for it to be that way, so i just left it alone (just in case someone had already built their CSV parsing from us to expect a certain row order; one of those things that was arbitrary during early early dev, but is now unfortunately locked in stone)